MojioSDK/Client/AuthClient.swift

```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```

Comment:
I would not add print statements in the SDK. 3rd party developers will not want to see such a vast amount of (potentially meaningless) information in their consoles. It's fine for debugging locally, but take it out of the SDK.